logfire/variables/remote.py

```
@@ -300,10 +314,15 @@ def _worker(self):
 
             self.refresh(force=force)
 
+            # Gap 3: add uniform +/-10% jitter to the wait so that fleets deployed
+            # simultaneously don't poll in lockstep.
+            base_interval = self._polling_interval.total_seconds()
+            wait_timeout = base_interval * random.uniform(0.9, 1.1)
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)